containerd-shim/process.go

```diff
@@ -132,7 +132,8 @@ func (p *process) start() error {
 		)
 	} else if p.checkpoint != nil {
 		args = append(args, "restore",
-			"--image-path", filepath.Join(p.checkpointPath),
+			"--image-path", p.checkpointPath,
+			"--work-path", filepath.Join(p.checkpointPath, "criu.work", "restore-"+time.Now().Format(time.RFC3339)),
```

Comment:
They'll be deleted whenever you delete the checkpoint. Strictly speaking, we don't *need* to create a new directory each time, but since you can restore the same checkpoint multiple times, the previous restore log would be overwritten if we don't do it this way.